client/my-sites/plugins/plans/index.tsx

```diff
@@ -60,10 +70,8 @@ const Plans = ( { intervalType }: { intervalType: 'yearly' | 'monthly' } ) => {
 					basePlansPath="/plugins/plans"
 					site={ selectedSite }
 					intervalType={ intervalType }
-					selectedFeature={ FEATURE_INSTALL_PLUGINS }
 					selectedPlan={ PLAN_BUSINESS }
-					shouldShowPlansFeatureComparison
-					isReskinned
+					planTypes={ [ currentPlanType, TYPE_BUSINESS, TYPE_ECOMMERCE ] }
```

Comment:
@candy02058912 and I tried this yesterday, it won't show the free plan while planTypeSelector is set to "interval" but setting this to anything else also hides the price toggle switch.

```suggestion
					planTypeSelector="anything-but-interval" // todo: replace this workaround to enable showing free plans
					planTypes={ [
						// unique to remove duplicate business/ecom plan types
						...new Set( [
							currentPlanType,
							TYPE_BUSINESS,
							TYPE_ECOMMERCE,
						] ),
					] }
```

Candy's original PR [had a bypass](https://github.com/Automattic/wp-calypso/pull/67891/files?w=1#diff-2545c62593dd002397248377b1c0feb643bd2acc7694054ce094997b1c029aceR405-R410) for [this code](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/plans-features-main/index.jsx#L395-L399). Guessing that's what we'll need?

Reply:
Decided to use the flowName this time instead of introducing a new `isInMarketplace` prop.